app/experimenter/nimbus-ui/src/components/TableSummary/index.tsx

```diff
@@ -0,0 +1,153 @@
+                    return probeSet.name;
+                  })
+                  .join(", ")
+              : notSet("Probe sets")}
+          </td>
+        </tr>
+        <tr>
+          <th className="font-weight-bold">Audience</th>
+          <td>
+            <span data-testid="experiment-target">
+              {displayConfigLabelOrNotSet(
+                "Target audience",
+                experiment.targetingConfigSlug,
+                targetingConfigSlug,
+              )}
+              ,{" "}
+            </span>
+            <span data-testid="experiment-channels">
+              {experiment.channels && experiment.channels.length
+                ? experiment.channels
+                    .map((expChannel) =>
+                      displayConfigLabelOrNotSet(
+                        "channel",
+                        expChannel,
+                        channels,
+                      ),
+                    )
+                    .join(", ")
+                : notSet("channel")}
+              {", "}
+            </span>
+            <span data-testid="experiment-ff-min">
+              {displayConfigLabelOrNotSet(
+                "Firefox minimum version",
+                experiment.firefoxMinVersion,
+                firefoxMinVersion,
+              )}
+            </span>
+            <span className="d-block">
+              <span data-testid="experiment-population">
+                {experiment.populationPercent
+                  ? `${experiment.populationPercent}% of population`
+                  : notSet("Population percentage")}
+              </span>
+              <span data-testid="experiment-total-enrolled">
+                {" "}
+                totalling{" "}
+                {experiment.totalEnrolledClients
+                  ? `${experiment.totalEnrolledClients.toLocaleString()} expected enrolled clients`
+                  : notSet("expected enrolled clients")}
+              </span>
+            </span>
+          </td>
+        </tr>
+        <tr>
+          <th className="font-weight-bold">Duration</th>
+          <td>
+            <span data-testid="experiment-duration">
+              {displayDaysOrNotSet(
+                "Proposed duration",
+                experiment.proposedDuration,
+              )}{" "}
+            </span>
+            <span data-testid="experiment-enrollment">
+              over an enrollment period of{" "}
+              {displayDaysOrNotSet(
+                "proposed enrollment",
+                experiment.proposedEnrollment,
+              )}
+            </span>
+          </td>
+        </tr>
+      </tbody>
+    </Table>
+  );
+};
+
+// type displayConfigOptionsProps =
+//   | Pick<getConfig_nimbusConfig, "firefoxMinVersion">
+//   | Pick<getConfig_nimbusConfig, "channels">
+//   | Pick<getConfig_nimbusConfig, "targetingConfigSlug">;
+
+const displayConfigLabelOrNotSet = (
+  description: string,
+  value: string | null,
+  options: any,
```

Comment:
@jodyheavener this and the object in `options.find` is where I know I need some TypeScript love. I kind of wonder if setting this to a generic array of objects is the best bet here (there's probably a prettier solution than what I have commented out regardless)... Open to any suggestions here. :)